torch/csrc/jit/passes/peephole.cpp

```diff
@@ -228,6 +228,10 @@ void PeepholeOptimizeImpl(Block* block, bool addmm_fusion_enabled) {
           WithInsertPoint guard(node);
           IValue ival(sizes);
           auto const_sizes_val = node->owningGraph()->insertConstant(ival);
+          GRAPH_UPDATE(
+              *node,
+              " (x.size()) is replaced with ",
+              const_sizes_val->debugName());
```

Comment:
could you please use `getHeader(node)` instead of `*node`. Otherwise, looks good!

Reply:
Done~

Should I replace all the previous style `*node` with `getHeader(node)`? 😀

Reply:
sure, that would be great!